justfile

```diff
@@ -12,6 +12,8 @@ public_regtest_coordinator := "03507b924dae6595cfb78492489978127c5f1e3877848564d
 public_regtest_esplora := "http://35.189.57.114:3000"
 public_coordinator_http_port := "80"
 
+local_ip := "10.60.173.164"
```

Comment:
we could probably retrieve this value from the shell instead of hardcoding it? 😅 I don't think it will work for anyone else except yourself on a particular network.

(see run-local suggestion for one way of achieving it)